### PR TITLE
Select default flag color for map image

### PR DESCRIPTION
**Pull request by rudivanhierden**
Related ticket: https://trello.com/c/x14bSEuQ/643-standaardvlag-optie-toevoegen-aan-global-settings-voor-de-vlag-op-de-google-maps-image

### CODE REVIEW

File: lib/modules/apostrophe-global/index.js [hunk 1]
```diff
@@ -182,6 +182,13 @@ module.exports = {
       label: 'Selecteer een polygon',
       choices: polygonsFormatted
     },
+    {
+      name: 'mapImageFlag',
+      type: 'select',
+      label: 'Welke kleur vlag moet er getoond worden op de plattegrond in het begroten overzicht en op de planpagina?',
+      choices: flagChoices,
```
Comment (stijnvandervegt) [inline lib/modules/apostrophe-global/index.js]: I don't see where the variable `flagChoices` is set

File: lib/modules/apostrophe-pages/index.js [hunk 2]
```diff
@@ -99,7 +99,8 @@ module.exports = {
                  if (idea.location && idea.location.coordinates[0] && idea.location.coordinates[1]) {
                    let lat = idea.location.coordinates[0];
                    let lng = idea.location.coordinates[1];
-                   idea.locationMapImage = `https://maps.googleapis.com/maps/api/staticmap?center=${lat},${lng}&zoom=15&size=380x214&maptype=roadmap&markers=icon:${appUrl}/img/idea/flag-red.png|${lat},${lng}&key=${googleMapsApiKey}`;
+                   let flag = req.data.global.mapImageFlag ? 'flag-' + req.data.global.mapImageFlag : 'flag-red';
+                   idea.locationMapImage = `https://maps.googleapis.com/maps/api/staticmap?center=${lat},${lng}&zoom=15&size=380x214&maptype=roadmap&markers=icon:${appUrl}/img/idea/${flag}.png|${lat},${lng}&key=${googleMapsApiKey}`;
```
Comment (stijnvandervegt) [inline lib/modules/apostrophe-pages/index.js]: Is this the only place where we want to set the mapImageFlag? 
What about the flags in the map-widget?

File: lib/modules/apostrophe-global/index.js (hunk 1 above)
Comment (rudivanhierden) [reply lib/modules/apostrophe-global/index.js]: It was already defined in the file, see line 26

File: lib/modules/apostrophe-pages/index.js (hunk 2 above)
Comment (rudivanhierden) [reply lib/modules/apostrophe-pages/index.js]: The flags in the map-widget depend on a lot more factors, for instance the theme of the idea and the status. The map image does not depend on any logic, so it should be sufficient to select this through the global settings.

**Comment by rudivanhierden** [comment]
PR made

**Pull request closed without merging**